### ISSUE TITLE
Can not work with quasar(v2) SSR

### ISSUE THREAD

**Issue by DefectingCat** [XiongAmao/vue-easy-lightbox#73]
I use import to import VueEasyLightbox  on SFC.

```vue
<template>
  <q-page class="row items-center justify-evenly">
    <vue-easy-lightbox
      escDisabled
      moveDisabled
      :visible="visible"
      :imgs="imgs"
      :index="index"
      @hide="visible = !visible"
    ></vue-easy-lightbox>
    <q-btn @click="visible = !visible">test</q-btn>
    {{ $route.params }}
  </q-page>
</template>

<script lang="ts" setup>
import { ref } from 'vue';
import VueEasyLightbox from 'vue-easy-lightbox';

const imgs = ref([
  { title: 'test img', src: 'http://via.placeholder.com/350x150' },
  'http://via.placeholder.com/350x150',
]);
const index = ref(1);

const visible = ref(false);
</script>

<style scoped lang="scss"></style>
```

At the beginning, it works. but if refresh the page with VueEasyLightbox  . page will crash with error follow:

```
Couldn't resolve component "default" at "/shots/:platform"
```

Here is my routes:

```ts
const routes: RouteRecordRaw[] = [
  {
    path: '/',
    component: () => import('layouts/MainLayout.vue'),
    children: [
      {
        path: '/',
        component: () => import('pages/Index.vue'),
      },
      {
        path: '/test',
        component: () => import('pages/Test.vue'),
      },
      {
        path: '/shots/:platform',
        component: () => import('pages/shots/Shots.vue'),
        // component: Shots,
      },
      {
        path: '/videos/:platform',
        component: () => import('pages/videos/Videos.vue'),
      },
    ],
  },
  // Always leave this as last one,
  // but you can also remove it
  {
    path: '/:catchAll(.*)*',
    component: () => import('pages/Error404.vue'),
  },
];
```

And error on console is `Window not defined`, i guess maybe is use the window too early

```
ReferenceError: window is not defined
    at Object.<anonymous> (D:\Git\Tahiti\node_modules\vue-easy-lightbox\dist\vue-easy-lightbox.es5.common.min.js:1:29255)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at _require (D:\Git\Tahiti\node_modules\@quasar\ssr-helpers\lib\create-bundle.js:56:9)
    at Object.vue-easy-lightbox (render-app.js:1792:18)
    at __webpack_require__ (render-app.js:1827:42)
[Quasar Dev Webserver] /shots/ps4 -> error during render
Error: Couldn't resolve component "default" at "/shots/:platform"
    at D:\Git\Tahiti\node_modules\vue-router\dist\vue-router.cjs.js:2016:47
```

If you need more information. Here is [repository](https://github.com/DefectingCat/Tahiti).

btw, VueEasyLightbox  is very nice.

**Comment by DefectingCat** [comment]
On server side, window is not available. if need to use window, we must be use it on client side.
I see the source code use `if` statement to judge whether the window exists. It works, but refreshed page will get `window not defined`
I guess maybe is [stateful singletons problem](https://quasar.dev/quasar-cli/developing-ssr/writing-universal-code#avoid-stateful-singletons), 
I try to fix them with :

```ts
export const inBrowser = () => typeof window !== 'undefined'
```

and

```ts
      onMounted(() => {
        if (inBrowser()) {
          on(document, 'keydown', onKeyPress)
          on(window, 'resize', onWindowResize)
        }
      })

      onBeforeUnmount(() => {
        if (inBrowser()) {
          off(document, 'keydown', onKeyPress)
          off(window, 'resize', onWindowResize)
        }
      })
```

well, it doesn't work.

But i found solution for this case, use the async component and `<no-ssr>` tag:

```vue
<template>
  <q-page class="row items-center justify-evenly">
    <q-no-ssr>
      <vue-easy-lightbox
        escDisabled
        moveDisabled
        :visible="visible"
        :imgs="imgs"
        :index="index"
        @hide="visible = !visible"
      ></vue-easy-lightbox>
    </q-no-ssr>
    <q-btn @click="visible = !visible">test</q-btn>
    {{ $route.params }}
  </q-page>
</template>

<script lang="ts" setup>
import { ref, defineAsyncComponent } from 'vue';
// import VueEasyLightbox from 'vue-easy-lightbox';
const VueEasyLightbox = defineAsyncComponent(() => import('vue-easy-lightbox'));

const imgs = ref([
  { title: 'test img', src: 'http://via.placeholder.com/350x150' },
  'http://via.placeholder.com/350x150',
]);
const index = ref(1);

const visible = ref(false);
</script>

<style scoped lang="scss"></style>
```

 `<no-ssr>` tag (in Quasar is `<q-no-ssr>`, other framework has similar tag) and `defineAsyncComponent()` make sure the component is running on client side.

**Comment by XiongAmao** [comment]
I used `iconfont` and it exported `assets/svg/iconfont.js` that didn't detect `window` correctly. 
It was fixed in v1.2.1
